lib/propolis/src/hw/pci/topology.rs

```diff
@@ -267,30 +267,29 @@ impl Builder {
 
         let pio_bus = &machine.bus_pio;
         let mmio_bus = &machine.bus_mmio;
-        let acc_mem = machine.acc_mem.child();
-        let acc_msi = machine.acc_msi.child();
```

Comment:
Reviewer(s) please note:  Creating these children this way was a bug!  The bus(es) created in the topology spawned their own children from the local `acc_mem` and `acc_msi`.  Those locals are dropped when the function returns, effectively orphaning those buses.

Reply:
Yes, precisely.  (Ask me how I know)